core/util/merkle_patricia_trie.go

```diff
@@ -141,6 +142,13 @@ func (mpt *MerklePatriciaTrie) Insert(path Path, value MPTSerializable) (Key, er
 		return mpt.Delete(path)
 	}
 
+	if len(eval) > MPTMaxAllowableNodeSize {
+		msg := fmt.Sprintf("node exceeds maximum permissible size, path: %s", string(path))
```

Comment:
may be also specify the current length `len(eval)` and the maximum allowed limit `MPTMaxAllowableNodeSize` in the error message. So someone who reads the message of those limit.